go.sum

```diff
@@ -36,6 +36,8 @@ github.com/ciscoecosystem/mso-go-client v1.2.6 h1:CU8l0cs15WNJFHQLJA/hJA6H4/vgra
 github.com/ciscoecosystem/mso-go-client v1.2.6/go.mod h1:ffy+aqN7Zo7GZC8e0pSt+Kbc32wjqkc+zDD42Ug0lWw=
 github.com/ciscoecosystem/mso-go-client v1.3.0 h1:nu5au4Q+3mu3wgRlfiystAclt1sYbjCHPf4tFe4FBe0=
 github.com/ciscoecosystem/mso-go-client v1.3.0/go.mod h1:ffy+aqN7Zo7GZC8e0pSt+Kbc32wjqkc+zDD42Ug0lWw=
```

Comment:
Can you remove the previous versions of mso-go-client in this file as they are not needed? (Anything before mso-go-client 1.4.0)?

Reply:
done